zc_install/includes/nginx_conf/ngx_conf_http.txt

```diff
@@ -0,0 +1,34 @@
+##############################################################
+##############################################################
+##############################################################
+### 		NGINX CONFIGURATION FOR ZEN CART
+###
+### Copy and paste into ‘http’ section of nginx conf file
+### IMPORTANT: 
+###    - Enables optional caching in serve section
+###    - Remember to uncomment relevant lines in server section
```

Comment:
It seems you've removed the `server` section lines to be `uncommented`. Was that intentional? Or should this line be removed if there's no longer a need to uncomment anything?